pages/common/uuencode.md

```diff
@@ -0,0 +1,16 @@
+# uuencode
+
+> Encode binary files into ASCII for transport via mediums that only support simple ASCII encoding.
+> More information: <https://manned.org/uuencode>.
+
+- Encode a file and print the result to stdout:
+
+`uuencode {{path/to/input_file}} {{output_file_name_after_decoding}}`
+
+- Encode a file and and write the result to a file:
+
+`uuencode -o {{path/to/output_file}} {{path/to/input_file}} {{output_file_name_after_decoding}}`
```

Comment:
If you already specify a output file, why do you have to specify the output file name separately?

Reply:
`path/to/output_file` is the file that the encoded data should be written to.  `output_file_name_after_decoding` is the name of the file that the decoded data will be written to when the file is decoded.  Makes sense?

Reply:
You could, and that's probably what the user would do.  But how should I indicate that?

Reply:
It's fine how you documented it, I was just wondering why the command works that way.

Reply:
It seems kinda dumb, I agree.